src/api/web.js

```diff
@@ -247,21 +254,17 @@ class WebInterface {
     const id = directoryRDF.sym(graphName);
     directoryRDF.setId(graphName);
 
-    let existingRDF = false;
     try {
       await directoryRDF.nowOrWhenFetched();
-      existingRDF = true;
     } catch (e) {
       // ignore no ID set in case nothing has been added yet
       if (e.code !== errConst.MISSING_RDF_ID.code) {
         throw new Error({ code: e.code, message: e.message });
       }
     }
 
-    if (!existingRDF) {
-      directoryRDF.add(id, vocabs.DCTERMS('title'), directoryRDF.literal(`${PUBLIC_CONTAINER} default container`));
-      directoryRDF.add(id, vocabs.DCTERMS('description'), directoryRDF.literal('Container to keep track of public data for the account'));
-    }
+    directoryRDF.add(id, vocabs.DCTERMS('title'), directoryRDF.literal(`${PUBLIC_CONTAINER} default container`));
```

Comment:
There's no PUT penalty for re-adding this if already existing, correct?

Reply:
No, remember this is jut in memory, in the triple store handled in memory by rdflib.js, an from my research the tripple already exists then nothing happens. After we finished adding all triple to the in memory graph store, we serialise it for sending it to the MD, and that's where we effectively merge the graphs, i.e. in the RDF.commit function.